files/consul-acl

```diff
@@ -0,0 +1,133 @@
+#!/bin/sh
```

Comment:
Why `sh` instead of `ruby`?

Reply:
Didn't write it or re-evaluate it, not sure.

Reply:
It was originally a much simpler shell script